allennlp/predictors/open_information_extraction.py

```diff
@@ -60,6 +60,72 @@ def make_oie_string(tokens: List[Token], tags: List[str]) -> str:
 
     return " ".join(frame)
 
+def get_predicate_indices(tags: List[str]) -> List[int]:
+    """
+    Return the word indices of a predicate in BIO tags.
+    """
+    return [ind for (ind, tag)
+            in enumerate(tags) if 'V' in tag]
+
+def get_predicate_text(sent_tokens: List[Token], tags: List[str]) -> str:
+    """
+    Get the predicate in this prediction.
+    """
+    return " ".join([sent_tokens[pred_id].text
+                     for pred_id in get_predicate_indices(tags)])
+
+def check_predicates_subsumed(tags1: List[str], tags2: List[str]) -> bool:
```

Comment:
I don't like the `check` in the name, it makes it read awkward when you use it later. I'd call it something like `predicates_are_subsumed`